components/dashboard/src/settings/Teams.tsx

```diff
@@ -0,0 +1,767 @@
+            <p className="pb-4 text-gray-500 text-base">Invite members to the team using the URL below.</p>
+
+            <div className="flex flex-col space-y-2">
+                <label htmlFor="inviteUrl" className="font-medium">Invite URL</label>
+                <div className="w-full relative">
+                    <input name="inviteUrl" disabled={true} readOnly={true} type="text" value={getInviteURL()} className="rounded-md w-full truncate pr-8" />
+                    <div className="cursor-pointer" onClick={() => copyToClipboard(getInviteURL())}>
+                        <img src={copy} title="Copy Invite URL" className="absolute top-1/3 right-3" />
+                    </div>
+                </div>
+                <p className="pb-4 text-gray-500 text-xs">Use this URL to join this team</p>
+            </div>
+
+        </div>
+        <div className="flex justify-end mt-6">
+            <button className={"ml-2"} onClick={() => props.onClose()}>Close</button>
+        </div>
+    </Modal>);
+}
+
+function AddMembersModal(props: {
+    sub: TeamSubscription,
+    onBuy: (plan: Plan, quantity: number, sub: TeamSubscription) => void,
+    onClose: () => void
+}) {
+
+    const [quantity, setQuantity] = useState<number>(5);
+
+    const [expectedPrice, setExpectedPrice] = useState<string>("");
+
+    useEffect(() => {
+        const plan = getPlan();
+        const expectedPrice = quantity * plan.pricePerMonth;
+        setExpectedPrice(`${expectedPrice}${Currency.getSymbol(plan.currency)}`);
+    }, [quantity])
+
+    const getPlan = () => {
+        return Plans.getAvailableTeamPlans().filter(p => p.chargebeeId === props.sub.planId)[0];
+    }
+
+    return (<Modal visible={true} onClose={props.onClose}>
+        <h3 className="pb-2">Add Members</h3>
+        <div className="border-t border-b border-gray-200 mt-2 -mx-6 px-6 py-4">
+            <p className="pb-4 text-gray-500 text-base">Add members to the team.</p>
+
+            <div className="flex flex-col space-y-2 pb-4">
+                <label htmlFor="quantity" className="font-medium">Members</label>
+                <select name="quantity" value={quantity} className="rounded-md w-full border-2 border-gray-400"
+                    onChange={(e) => setQuantity(e.target.value as any)}>
+                    {Array(20).fill(1).map((_, index) => index + 1).map(n => (
+                        <option key={`quantity-${n}`} value={n}>{n}</option>
+                    ))}
+                </select>
+            </div>
+
+            <div className="flex rounded-md bg-gitpod-kumquat-light p-3">
+                <img className="w-4 h-4 mx-2 my-auto" src={exclamation} />
+                <span className="text-red-600">Total: {expectedPrice} per month</span>
+            </div>
+
+        </div>
+        <div className="flex justify-end mt-6">
+        <button className={"ml-2"} onClick={() => props.onBuy(getPlan(), quantity, props.sub)}>Continue to Billing</button>
```

Comment:
**thought**: We could also have a secondary Cancel button on the left of this button as in some other areas of the product. Feel free to ignore this for this milestone.